app/src/main/java/ai/arena/hello/hellokotlinformat/MainActivity.kt

```diff
@@ -13,6 +13,7 @@ import com.google.android.material.snackbar.Snackbar
 
 class MainActivity : AppCompatActivity() {
 
+  //test commit without installing the hooks with command line ` pre-commit install`
   private
```

Comment:
Because `pre-commit install` is not installed, hence the format rule will not be enforced.